src/ORM/Connect/TempDatabase.php

```diff
@@ -23,6 +24,13 @@ class TempDatabase
      */
     protected $name = null;
 
+    /**
+     * Optionally remove the test DB when the PHP process exits
+     *
+     * @var boolean
+     */
+    protected static $teardown_on_exit = true;
```

Comment:
If you want this to be a config prop it'll need to be private

Reply:
whoops thanks :)

Reply:
just looping back in @robbieaverill changes made :)